scripts/prepare-espresso-crypto-helper

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Use `id -u` to check if the script is being run as root.
+# If not, prepend `sudo` to commands.
+SUDO=$(if [ "$(id -u)" -ne 0 ]; then echo sudo; fi)
+
+arch=$(uname -m)
+os=$(uname -s)
+
+if [ "$arch" = "arm64" ] || [ "$arch" = "aarch64" ]; then
+    if [ "$os" = "Darwin" ]; then
+        triple="aarch64-apple-darwin.dylib"
+    else
+        triple="aarch64-unknown-linux-gnu.so"
+    fi
+elif [ "$arch" = "x86_64" ]; then
+    if [ "$os" = "Darwin" ]; then
+        triple="x86_64-apple-darwin.dylib"
+    else
+        triple="x86_64-unknown-linux-gnu.so"
+    fi
+else
+    echo "ERROR: $arch is not supported"
+    exit 1
+fi
+
+go get github.com/EspressoSystems/espresso-network/sdks/go
+go get github.com/EspressoSystems/espresso-network/sdks/go/download
+
+$SUDO go run github.com/EspressoSystems/espresso-network/sdks/go/download clean
```

Comment:
I dont think we should use sudo because it will cause issues with docker permissions. This is why I dont like this linking solution.

Reply:
Ohh I see, now that makes sense to me!!